.travis.yml

```diff
@@ -17,16 +20,16 @@ notifications:
 before_install:
   - "export DISPLAY=:99.0"
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sh -e /etc/init.d/xvfb start; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then scripts/travis-install-sdl2.sh ; fi
```

Comment:
please add a comment saying that this is temporary, and when it's no longer going to be needed